arbitrum-docs/launch-orbit-chain/timeboost-for-orbit.mdx

```diff
@@ -0,0 +1,91 @@
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `roundDurationSeconds`  | Duration of time that the sequencer will honor the express lane privileges for transactions signed by the current round’s express lane controller. Default: 60 seconds                                                                               | The larger this value, the more powerful and valuable controlling the express lane will be. Higher values may incentivize more participation and demand for the express lane time advantage.                                                  |
+| `auctionClosingSeconds` | Time before the start of the next round. The autonomous auctioneer will not accept bids during this time interval. Default: 15 seconds                                                                                                               | The larger the value, the more difficult it is for participants to predict the value of the future express lane round, and therefore, their ability and confidence in being able to place an accurate/fair bid for the time advantage. However, too small of a value may leave not enough time for the auctioneer to resolve the auction in a timely fashion during periods of high activity. |
+| `beneficiary`           | An address where proceeds from the Timeboost auction are sent to when `flushBeneficiaryBalance()` gets called on the auction contract. Default: An address controlled by the chain's owner                                                              | Can be any address, including the burn address, that the chain owner desginates or controls.                                                                                                                                                  |
+| `_biddingToken`         | Address of the token used to make bids in the Timeboost auction. It can be any `ERC-20` token, including the gas token of the network (assuming the token address chosen does not have fee-on-transfer, rebasing, transfer hooks, or otherwise non-standard `ERC-20` logic). Default: `WETH` | This asset should ideally be liquid and easily obtainable for your auction participants. Furthermore, the less volatile this asset is, the more consistent your auction will be.                                                              |
```

Comment:
```suggestion
| `_biddingToken`         | Address of the token used to make bids in the Timeboost auction. It can be any `ERC-20` token, including the gas token of the network, provided the chosen token address does not have fee-on-transfer, rebasing, transfer hooks, or otherwise non-standard `ERC-20` logic. Default: `WETH` | This asset should ideally be liquid and easily obtainable for your auction participants. Furthermore, the less volatile this asset is, the more consistent your auction will be.                                                              |
```